integreat_cms/cms/templates/pages/page_sbs.html

```diff
@@ -36,7 +36,9 @@ <h1 class="heading">
                                 {% endif %}
                             </button>
                         {% elif can_edit_page %}
-                            <button name="status" value="{{ REVIEW }}" class="btn">
+                            <button name="status"
```

Comment:
Currently we don't have ajax slug generation in the sbs form (probably it's a bug though)

Reply:
I have created a separate issue for that